src/main/java/org/mastodon/revised/model/mamut/trackmate/MamutExporter.java

```diff
@@ -681,7 +681,9 @@ public static void main( final String[] args ) throws IOException
 		final String bdvFile = "samples/datasethdf5.xml";
 		final MamutProject project = new MamutProject( new File( projectFolder ), new File( bdvFile ) );
 		final Model model = new Model();
-		model.loadRaw( project );
+		final MamutProject.ProjectReader reader = project.openForReading();
+		model.loadRaw( reader );
+		reader.close();
```

Comment:
Could we make this compatible with the `try-with-resources` construct?
We pass input and output streams around but they are still `AutoCloseable` so maybe something like
```java
try(final MamutProject.ProjectReader reader = project.openForReading())
{
   model.loadRaw( reader );
}
```
would normally work.

Reply:
Sounds good!

Reply:
Let's go!